tests/network-tests/src/nicaea/tests/workingGroup/impl/workingGroupModule.ts

```diff
@@ -509,3 +504,90 @@ export async function awaitPayout(apiWrapper: ApiWrapper, membersKeyPairs: Keyri
 export async function setMintCapacity(apiWrapper: ApiWrapper, sudo: KeyringPair, capacity: BN, module: WorkingGroups) {
   await apiWrapper.sudoSetWorkingGroupMintCapacity(sudo, capacity, module);
 }
+
+export async function expectLeadOpeningAdded(apiWrapper: ApiWrapper): Promise<BN> {
+  return apiWrapper.expectOpeningAdded();
+}
+
+export async function expectLeaderSet(
+  apiWrapper: ApiWrapper,
+  leaderAddress: string,
+  module: WorkingGroups
+): Promise<BN> {
+  const leadWorkerId: BN = await apiWrapper.expectLeaderSet();
+  const worker: Worker = await apiWrapper.getWorkerById(leadWorkerId, module);
+  const leaderApplicationId = (await apiWrapper.getApplicationsIdsByRoleAccount(leaderAddress, module))[0];
+  const application: Application = await apiWrapper.getApplicationById(leaderApplicationId, module);
+  assert(
+    worker.role_account_id.toString() === application.role_account_id.toString(),
```

Comment:
I think you can do:
```suggestion
    worker.role_account_id.eq(application.role_account_id)
```

If both are of type AccountId

Reply:
Done in https://github.com/Joystream/joystream/pull/1001